docs/en/enterprise-edition/content-collections/application-security/risk-management/monitor-and-manage-code-build/enforcement.adoc

```diff
@@ -99,160 +105,119 @@ For every failed scan result you can view the latest Pull Request (PR) of your r
++ 
+NOTES:
+
+* To add labels while creating policies, refer to xref:../../../governance/custom-build-policies/custom-build-policies.adoc[Custom Build Policies] 
+* To view labels assigned to Application Security policies: On the Prisma Cloud console, select *Governance*, policy type: *config*, policy subtype *Build*. View the label associated to a policy under the Labels column in a row corresponding to the policy
+* To edit labels, select the *Edit* icon in the policy row
 
 
 [.task]
 
 [#add-an-exception-to-enforcement]
-=== Add an Exception to Enforcement
+=== Add an Exception
 
-To ensure your focus is only on critical issues and you receive violation notifications on important repositories, you can add an exception to the Enforcement.
+To ensure focus on critical issues and receive violation notifications for important repositories, add exceptions to Enforcement rules. You can add an exception for each code category that is applicable, but this only applies to repositories on which you have the required permissions. The exception configuration runs in addition to the default enforcement configurations.
+Configuring an exception includes defining the scope of the exception, and specifying the repositories code categories and run rules that the exception will apply to.
 
 [.procedure]
 
-. <<access-enforcement,Access>> enforcement.
+. <<access-enforcement,Access Enforcement>> settings.
 
-. Add an exception to enforcement.
-.. Select *Add exception*.
+. Select *Add Exception*.
 +
 image::application-security/enfor-6.png[]
 
-. Configure exception parameters.
-.. Add *Description* to the new exception.
-.. Select the repositories you want to add the exception.
+. Add a *Description* to the new exception.
+. Select the repositories you want to add the exception.
 +
 NOTE: You can only view repositories that you own.
 
-.. Select a code category.
-
-.. Select the severity threshold corresponding to the code category.
-+
-You can choose to continue modifying other code categories or conclude with a single modification.
+. Modify a severity threshold corresponding to the required code category/ categories.
 
-.. Select *Save* to save the exception with the parameters.
+. Select *Save*.
 +
-Here is an example of an exception.
+EXAMPLE
 +
 image::application-security/enfor-21.png[]
 +
-All exception configurations are listed on *Enforcement*.
+NOTE: All exception configurations are listed on *Enforcement*.
 +
 image::application-security/enfor-22.png[]
-+
-You can optionally choose to edit or delete an existing exception.
-+
-** To edit an exception, hover over the Exception and then select *Edit* to configure the parameters. Select *Save* to save the modification to the exception.
-+
-image::application-security/enfor-10.png[]
-+
-** To delete an exception select *Edit* and then select *Delete this exception*.
-+
-image::application-security/enfor-11.png[]
 
-[.task]
+==== Editing/Deleting an Exception
```

Comment:
We generally do not use / in headings. I would suggest changing this to "Modifying and Exception" to avoid this issue.

Reply:
Changed to 'Modifying or Deleting an Exception' - this is due to the fact that 'modify' does not include 'delete'